### ISSUE TITLE
Clarification 1.2.5: Audio Description versus 1.2.7 Extended Audio Description 

### ISSUE THREAD

**Issue by PICARDBRB** [w3c/wcag#2479]
For a video with a text media alternative under 1.2.3 Level A, and the author must meet Level AA criterion, and where the audio track of a prerecorded video does not have sufficient pauses for audio description does the content meet 1.2.5 because the author cannot technically embed the audio descriptions as described in the video?   **OR** must the author  use a technique to create a second audio track with descriptions,  the author would need to apply a technique that meets 1.2.7? 

1.2.5 Audio description, describes the requirement to provide audio description for all prerecorded video content in synchronized media.  The intent is to include audio in the existing pauses in the dialogue although this is not stated explicitly in the Success Criterion statement.  

1.2.7 Extended Audio Description: Where pauses in foreground audio are insufficient to allow audio descriptions to convey the sense of the video, extended audio description is provided for all prerecorded video content in synchronized media.

It appears to me that if the author must create a second audio track (and there is no choice) to meet 1.2.5, then what is the purpose of 1.2.7?

Thanks for your help

**Comment by bruce-usab** [comment]
> For a video with a text media alternative under 1.2.3 Level A

Providing a text alternative for time-based media (or not) is not a factor with the situation you describe, because the AA requirement is to provide description.  It improves accessibility, but text the equivalent is above and beyond what 1.2.5  requires.

> ...and the author must meet Level AA criterion, and where the audio track of a prerecorded video does not have sufficient pauses for audio description

The AA SC 1.2.5 requirement is for audio description.  Per the WCAG2 definition for audio description, emphasis added, <q>narration is added during existing **pauses in dialogue**.</q>.

You mention SC 1.2.7, but that might not be the only option.  Does the video require?
> narration added to the soundtrack to describe important visual details that cannot be understood from the main soundtrack alone

If the video is using, for example, orchestral music to set the mood &mdash; then to meet 1.2.5 then the default background audio level needs to be lowered, and voice narration dubbed over it.

On the other hand, if there is constant talking/narration/dialogue in the video, it is probably okay (from a 1.2.5 perspective) because the literal textual requirement of the SC is being met.

> does the content meet 1.2.5 because the author cannot technically embed the audio descriptions as described in the video? 

The content meets 1.2.5 if a version with audio description is available.  That can be the main/default version.  That can be a version with additional narration (dubbed over pauses in dialog).  That can be a version using extended audio description.

It can be the case that there (1) are no pauses in dialog, but regardless there (2) are <q>important visual details that cannot be understood</q>.   For better or worse, this is not a failure against SC 1.2.5.  This situation is analogous to an image which has 
 not-so-great alt attribute value.  WCAG2 SC tend to be binary, and quality not so easily addressed by SC only.  How descriptive must a descriptive identification need to be?  Q:  How great does the narration have to be for the narration to count as audio description?  (A:  Not that great!  Quality, per se, is not part of the normative SC text.)

Better accessibility is almost certainly provided by meeting AAA 1.2.7, but there are not circumstances where meeting AA 1.2.5 _requires_ 1.2.7.

> Should 1.2.7 be combined with 1.2.5 and techniques supplied include both embedding audio description in existing dialog pauses or through a 2nd version of the media with the descriptions?

No.

**Comment by PICARDBRB** [comment]
Thank you for additional details.  Appreciate if we can dig into 1.2.5 a bit further to clarify understanding and application?

Example video: https://www.w3.org/WAI/perspective-videos/notifications/

_Does the video require narration to describe important visual details that cannot be understood from the main soundtrack alone_: Yes

_On the other hand, if there is constant talking/narration/dialogue in the video, it is probably okay (from a 1.2.5 perspective) because the literal textual requirement of the SC is being met_. : Can you confirm the example video would be sufficient to meet 1.2.5 without an additional sound track with audio description?

_It can be the case that there (1) are no pauses in dialog, but regardless there (2) are important visual details that cannot be understood. For better or worse, this is not a failure against SC 1.2.5._ : What is the testable criteria that would fail 1.2.5?  

Thanks for your help.

**Comment by bruce-usab** [comment]
Thank you @PICARDBRB for that great cite!  I had watched that video before, but until now, had missed/overlooked the Enable Audio Description button.  I watched it just now, before reading your comment.  It is a perfect example to chew on here!

> Does the video require narration to describe important visual details that cannot be understood from the main soundtrack alone?

No, it does not.  With my recent re-watch, I did think it was a shame not to mention the clock hands / stuck cuckoo.  I presumed the author regarded those as pure decoration (eye candy).  In my head, I am thinking that I would have taken a little more time with the voice over.  It seemed like a rather obvious missed opportunity &mdash; then I noticed the Enable Audio Description button.

> Can you confirm the example video would be sufficient to meet 1.2.5 without an additional sound track with audio description?

As a neutral/disinterested auditor, yes, absolutely.  I make that argument in part because, before watching/listening to the Audio Described version just now, _I totally missed that the error messages transformed_.  (In my defense, that animation is pretty fast/subtle.)

The default soundtrack is sufficient and complete (the whole message, all the information), but without the additional narration it is easy to miss stuff (visual details, not understood in the main soundtrack, but not necessarily important and not itself informative).

Since there are no pauses in dialog, I don't see how one can argue that the non-AD version fails 1.2.5.

The additional narration does exactly what theatrical quality AD is meant for:  verbal equivalent of visual presentation.

I think this is also a good example of how AD can benefit just about anyone, and not just someone who is blind.

I will also note the that the better version, with the extra narration, is almost twice as long.  It is a good example of how effective narration could/should be considered.  It does also model a nice implementation of 1.2.7 and Extended Audio Description.  OTOH, the shorter version is not very effective, because it is too fast.

A more traditional implementation of 1.2.7 won't have extra video to put under the additional narration, so the video image just does a freeze-frame.  With this particular sample from the WAI Perspectives Videos, the AD version is 1:54 while the default version is 1:06.  With AD narrator calling attention to the better error messages, and the AD version giving that effect more screen time, it is not surprising that the AD version is better!

**Comment by PICARDBRB** [comment]
First of all, I would like to compliment on all the Perspective videos...these videos and presentation in the Web Page along with the player are Wonderful!...I often refer others to these video's and for demo/example purposes of Success Criterion.

If you can follow me on this dive a bit further.....same video:

_Since there are no pauses in dialog, I don't see how one can argue that the non-AD version fails 1.2.5._

I am concerned.....if the non-AD version does not fail 1.2.5 then it is Level AA conforming for 1.2.5 without providing a text media alternative.  Under 1.2.3, the important visuals are described for the media alternative in text.  Those important visuals are included in the media alternative at the bottom of the page.  I could argue, given the author chose to include the text equivalent for the description that is not part of the default audio track, it would fail 1.2.5 without the second audio track?

Thoughts?

**Comment by bruce-usab** [comment]
Yes, agreed, those Perspective Videos are fantastic.  Taking this opportunity to (again) HT to @shawna-slh and EO since stimulating exactly this sort of sidebar conversation is part of the intent!

- The non-AD version passes 1.2.3 twice, as both the transcript and AD version are provided.
- The non-AD version passes 1.2.5 since there is the the AD version.
- The AD version passes 1.2.3 because it is AD.  1.2.3 is either or.  1.2.3 provides the author a choice.
- The AD version passes 1.2.5 because it is AD.

It has been an ongoing concern that there is no requirement for providing a transcript in addition to AD, since that would better accessibility, but personally I think that is okay.  It is a compromise.

@PICARDBRB &mdash; I may have missed your argument.  If so, please continue your patience! 

And since videos are on my mind, I will also take the liberty to mention this new and also great set of videos:  https://adata.org/ocr-videos

**Comment by shawna-slh** [comment]
>  HT to @shawna-slh and EO

Thanks! HT to EOWG, @nitedog , others: [Acknowledgements](https://www.w3.org/WAI/perspective-videos/acknowledgements/)!

And I'll take this opportunity to plug the EOWG resource [Making Audio and Video Media Accessible](https://www.w3.org/WAI/media/av/), which advises going beyond the minimum requirements of WCAG. For example:
> Yes, descriptive transcripts are needed to meet the wide range of user needs.

Read more about [descriptive transcripts](https://www.w3.org/WAI/media/av/planning/#descriptive-transcripts).

**Comment by PICARDBRB** [comment]
Hi, Yes i agree with all of the points and understand why the video passes with enabled descriptions audio track:
The non-AD version passes 1.2.3 twice, as both the transcript and AD version are provided.
The non-AD version passes 1.2.5 since there is the the AD version.
The AD version passes 1.2.3 because it is AD. 1.2.3 is either/or; 1.2.3 provides the author a choice.
The AD version passes 1.2.5 because it is AD.

If the non-AD version did not have a transcript and did not have the "enable descriptions" audio track, you indicated it still passes 1.2.5 because of the _exception in 1.2.5 "since there are no pauses in dialog"._ 

At 1.2.3 I can choose Media alternative as text or provide AD, as a strict requirement without exception.  At 1.2.5 I can choose AD and if my media does not have pauses in dialog, I will not need to provide either.  It appears Level AA creates a gap due to the exception worded in the criterion.

In my opinion, Level A is higher by providing a strict requirement.  Would you consider adding a second Level AA criterion to close the gap?  

Thanks for the new video library reference.

**Comment by PICARDBRB** [comment]
Thanks @DavidMacDonald , updating the understanding would help. 
Have a wonderful day!
